packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/leaf-more-menu.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * WordPress dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+import {
+	addSubmenu,
+	chevronUp,
+	chevronDown,
+	moreVertical,
+} from '@wordpress/icons';
+import { DropdownMenu, MenuItem, MenuGroup } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import { BlockTitle, store as blockEditorStore } from '@wordpress/block-editor';
+
+const POPOVER_PROPS = {
+	className: 'block-editor-block-settings-menu__popover',
```

Comment:
The styles associated with this class name are written in the `block-editor` package.

We should avoid re-using classnames (especially across packages) as it can lead to unexpected side effects. I suggest we add a new classname specific to this file/package, and (if needed) copy the styles from the block editor over.

The same should be done in the other "leaf more dropdown" component

Reply:
After looking more into this, it doesn't look like this class achieved much in terms of styling (the `padding` override doesn't seem to have effect). I'll also look into other parts of the repo relying on this, and will potentially remove the class altogether